package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@moox/bs-react-helmet",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

```suggestion
  "version": "3.0.0",
```

Should be changed on release.